pkg/util/gendoc/gendoc.go

```diff
@@ -14,7 +14,8 @@ import (
 )
 
 const (
-	gendocTypeFlag = "type"
+	gendocDisableAutoGenTagFlag = "disable-auto-gen-tag"
```

Comment:
not sure we need it, but dont mind, mb @roman-khimov has some thoughts

Reply:
I think we do because otherwise we can't have consistent output each time we run the generator.

Reply:
i meant if we need it to be possible to change via separate command flag